integration-tests/tests/workspace-dashboard.js

```diff
@@ -8,9 +8,8 @@ const { registerTest } = require('../utils/jest-utils')
 const { withUserToken } = require('../utils/terra-sa-utils')
 
 
-const azureWarning = 'It is a violation of US Federal Policy to store any Unclassified Confidential Information (ie FISMA, FIPS-199, etc.) ' +
-  'in this platform at this time. Do not put this data in this platform unless you are explicitly authorized to by the manager of the Dataset ' +
-  'or you have your own agreements in place.'
+const azureWarning = 'Do not store Unclassified Confidential Information in this platform, it violates US Federal Policy (ie FISMA, FIPS-199, etc), unless explicitly authorized by the dataset manager or governed by your own agreements.'
```

Comment:
Why is this text defined twice? Should the tests be able to reference a var defined elsewhere instead of re-defining it?

Reply:
Yeah I also hate that. I had extracted it as a const but then the integration tests were failing

Reply:
Let me take another look and see if I can make it work

Reply:
Booo hisss

Reply:
It looks like all the imports used for integration tests are pulled from `integration-helpers` or `integration-utils` and I don't think we want the warning text to live there, so it might just have to be duplicated for now :/

Reply:
pulling ui text strings out to a more sharable form that the integration tests can get at should probably be considered under the umbrella of implementing an off-the-shelf solution for internationalization (i18n).... so yeah... in other words... duplicate it for now and treat it as a fire alarm to update the integration test.

Reply:
@msilva-broad Thanks for the direction on that! I was planning to make a ticket for follow-up work. I think there are a couple of ways we could approach this: a) make it easier to pull text strings into integration tests, b) refactor this check out of the integration tests and into a unit test c) all of the above. Do you want me to make a ticket for either/both of these?